conda_smithy/templates/README.md.tmpl

```diff
@@ -71,10 +71,22 @@ Current build status
 ====================
 {%- set appveyor_badge_name = github.repo_name.replace('_', '-') %}
 {%- set appveyor_url_name = appveyor_badge_name.replace('.', '-') %}
-
+{# #}
+{%- if "circle" in ci_enabled %}
 Linux: [![Circle CI](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}}.svg?style=svg)](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}})
+{%- else %}
+Linux: ![](https://cdn.rawgit.com/{{github.user_or_org}}/{{github.repo_name}}/master/ci_support/disabled.svg)
```

Comment:
You are right, but honestly I'm not worried about it. We can always hold a copy of the SVG in the location for a period of time to give us grace on moving the feedstocks over. I don't think we need to bikeshed this one - personally I'd rather be able to move quickly on updating the SVG than worry about pointing at a specific release. Either way works for me though - you can have the final call on it.